src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -43,12 +43,12 @@ public Datastore() {
    * 
    *  @return true if a user in Datastore, false otherwise.
```

Comment:
Since the method returns an Entity now, I would change the method header just for reference, and perhaps change the method name to findUser (or something along those lines), just so it doesn't seem that a boolean should be returned!

Reply:
Good catch, I forgot to update it after changing it.